cfpb/cfgov-refresh
# Fix timezones

This reverts the code @kurtw introduced in https://github.com/cfpb/cfgov-refresh/pull/1622 (sorry Kurt!) but instead handles the issue at the time the datetime is saved. 

cc @kave @rosskarchner 

UPDATE:
- I added a script to update the existing datetimes in the db. 
- I did not update `ItemIntroduction.date` because we don't have any DemoPages yet. It also doesn't seem necessary to update the `HomePage.latest_updates.date` since it seems like we will be updating that anyway. Otherwise I am updating all Datetimes. Dates are always naive so we don't have to worry about those.

ANOTHER UPDATE:

**Testing instructions:**

- Start with a refresh db. 
- Load up a page like [http://localhost:8000/about-us/events/spring-2016-credit-union-advisory-council-meeting-washington-dc/](http://localhost:8000/about-us/events/spring-2016-credit-union-advisory-council-meeting-washington-dc/) and observe the times there
- Run `python cfgov/manage.py runscript fix_datetimes`. 
- Refresh the above page & make sure these changes propagated - times should be 4 hours ahead in this case. 
- Confirm these times are the same as in [www](http://files.consumerfinance.gov/f/201603_cfpb_meeting-agenda_cuac.pdf)

- Go to [http://localhost:8000/the-bureau/leadership-calendar/](http://localhost:8000/the-bureau/leadership-calendar/) and confirm these times match those in [www](http://www.consumerfinance.gov/leadership-calendar/) (Just pick out one or two)

- Create an event page or edit an event page & enter in a start time and an end time. Preview or publish the page and confirm it shows the same time you input. 
- Go into the shell and confirm the times are stored in UTC (and thus are 4 hours ahead)

- You can look at events before Daylight Savings (before March 13) and confirm those are actually 5 hours ahead of what's being displayed on the site.

Or if you think of other scenarios, feel free to try em instead!
